### ISSUE TITLE
Bug in button-on and button-off

### ISSUE THREAD

**Issue by fsedarkalex** [igorgladkov/rotini#23]
I have following config:

```
    Frame label="Licht An {widget:button-on}" {
      Switch item=sw_ledcontroller
    }
    Frame label="Licht Aus {widget:button-off}" {
      Switch item=sw_ledcontroller
    }
```

When using one of the buttons once none of both is working anymore until switching sitemaps or reloading the app.

(btw: Do you udate item states regularly?)

**Comment by igorgladkov** [comment]
Are those items defined with autoupdate=false config? Because there is a existing bug with those.
App listens to server push, so it should update immediately.

**Comment by fsedarkalex** [comment]
Switch sw_ledcontroller "LED-Hauptbeleuchtung [%s]" <switch> { rfxcom=">19088640.2:LIGHTING2.HOME_EASY_EU:Command" }

When it comes to push: I tried a normal light-switch and when switching it from my PC your widget does not get updated :( Even after minutes. When reloading sitemap it shows correct state again.

**Comment by igorgladkov** [comment]
What OpenHAB version do you have?

**Comment by fsedarkalex** [comment]
1.7.1

Am 10. Januar 2016 03:58:01 MEZ, schrieb Igor Gladkov notifications@github.com:

> What OpenHAB version do you have?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/igorgladkov/rotini/issues/23#issuecomment-170303803

## 

Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.

**Comment by fsedarkalex** [comment]
this issue is now alos fixed for me with latest update.. strange...

**Comment by igorgladkov** [comment]
@fsedarkalex this problem would be consequence of https://github.com/igorgladkov/rotini/issues/28, so if auto-update is working for you now, this is why this is not an issue anymore

**Comment by anordvall** [comment]
@igorgladkov I assume this also affects the mute, previous song and next song buttons in the music widget? At least the symptom is the same..

**Comment by igorgladkov** [comment]
yes, the same issue

**Comment by igorgladkov** [comment]
Fixed in 0.3.21
